README.md

```diff
@@ -96,6 +96,27 @@ docker image rm <image-id>
 
 There is no need to remove postgres and java or similar core images.
 
+### Idam Stub
+
+The service can be used to stub IDAM calls. It comes with a preconfigured stub IDAM user with some defaults IDAM roles.\
+When testing CCD it's very often needed to customise the roles list. For example to test how certain a feature behaves for a caseworker compared to a solicitor.\
+The following endpoint can be used to change the stubbed user info at runtime by posting the desired user info as JSON:
+
+```
+/idam-user
+```
+
+Example:
+
+```
+curl -X POST \
+--header 'content-type: application/json' \
+--data '{"email":"auto.test.cnp@gmail.com","roles":["caseworker-autotest1","caseworker-autotest2","caseworker-autotest3","caseworker","ccd-import"],"sub":"auto.test.cnp@gmail.com","uid":"7689","name":"CCD Auto Test (Stub2)","given_name":"CCD","family_name":"Auto Test (Stub2)"}' \
```

Comment:
Why not existing wire-mock API. http://wiremock.org/docs/api/#tag/Stub-Mappings/paths/~1__admin~1mappings/post
(we already have an example userinfo mapping file in the code).